app/src/main/kotlin/cz/covid19cz/erouska/jobs/AutoRestartJob.kt

```diff
@@ -0,0 +1,84 @@
+package cz.covid19cz.erouska.jobs
+
+import android.app.AlarmManager
+import android.app.PendingIntent
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import android.os.Handler
+import cz.covid19cz.erouska.BuildConfig
+import cz.covid19cz.erouska.service.CovidService
+import cz.covid19cz.erouska.utils.L
+import org.threeten.bp.ZonedDateTime
+
+class AutoRestartJob : BroadcastReceiver() {
+
+    companion object {
+        const val EXTRAKEY_START = "AUTO_RESTART_START"
+        const val EXTRAKEY_CANCEL = "AUTO_RESTART_CANCEL"
+
+        private const val ACTION = "EROUSKA_ALARM"
+
+        private val SCHEME = "erouska" + if (BuildConfig.FLAVOR == "dev") "-dev" else ""
```

Comment:
btw ten string si muzes dat do controls.xml a pak stejne jako s manifestem mit dve verze a podle flavoru ti context.getstring vrati tu spravnou.

Reply:
OK, zkusim.

Reply:
Nj, to je pěkný, jenže to to zase nemůžu dát do statickýho kontextu, protože tam nemam `Context`, ne?

Reply:
to ne..ale tak to neni potreba mit v companion objectu

Reply:
No jo, ale když to potřebuju na dvou různých místech, tak to je trochu smutný to nemít udělaný jen jednou :-D

Reply:
klidne to nech takhle 😃